Tests/GlueTestProject/GlueTestProject/GlueTestProject/TileGraphics/TiledMapSave.Serialization.cs

```diff
@@ -348,6 +351,20 @@ public int VisibleAsInt
             set { _visibleAsInt = value; }
         }
 
+        [XmlAttribute("offsetx")]
+        public float offsetX
```

Comment:
The XmlAttribute allows the class to match the XML with the name "offsetx". The property can be named differently, and should be capitalized to match C# standards. I know we don't always do this - I didn't write the code originally, but I'd like to have it be capitalized for all new code.